indico/modules/events/registration/legacy_notifications.py

```diff
@@ -0,0 +1,147 @@
+# This file is part of Indico.
```

Comment:
It is, but since I added stuff to the "new' notifications.py git probably decided to show it as a new file and a modification to the existing one